.zuul.yaml

```diff
@@ -1,8 +1,82 @@
 ---
+- nodeset:
+    name: stf-crc_extracted-ocp412
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-vexxhost
+      - name: crc
+        label: coreos-crc-extracted-2.19.0-xxl
+
+- nodeset:
+    name: stf-crc_extracted-ocp413
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-vexxhost
+      - name: crc
+        label: coreos-crc-extracted-2.28.0-xxl
+
+- nodeset:
+    name: stf-crc_extracted-ocp414
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-vexxhost
+      - name: crc
+        label: coreos-crc-extracted-2.30.0-xxl
+
+- job:
+    name: stf-base-2node
+    parent: podified-multinode-edpm-deployment-crc
+    abstract: true
+    required-projects:
+      - name: openstack-k8s-operators/dataplane-operator
+        override-checkout: main
+      - name: openstack-k8s-operators/infra-operator
+        override-checkout: main
+      - name: github.com/openstack-k8s-operators/openstack-operator
+        override-checkout: main
+      - name: openstack-k8s-operators/openstack-must-gather
+        override-checkout: main
+    pre-run:
+      - ci/pre-2node.yml
+    vars:
+      cifmw_deploy_edpm: false
+      podified_validation: true
+      cifmw_run_tests: false
+    extra-vars:
+      crc_ci_bootstrap_networking:
```

Comment:
Can you offer a few words on exactly why this config is needed? Is it related to the "Hacky-Patch machineNetwork range in run_ocp.sh" in the manage_ocp_cluster role in stf-v-c?

Reply:
Nope, this is based on a cookbook recipe in the CI framework. 

https://github.com/openstack-k8s-operators/ci-framework/blob/main/docs/source/cookbooks/zuul-job-nodeset.md